src/design-system/components/AnimatedText/AnimatedText.tsx

```diff
@@ -0,0 +1,76 @@
+import { Variants, motion } from 'framer-motion';
+import React from 'react';
+
+import { CHARACTER_TYPING_SPEED } from '~/entries/popup/pages/home/Points/utils';
+
+import { Text, TextProps } from '../Text/Text';
+
+type AnimationDirection = 'rightToLeft' | 'leftToRight';
+
+export const AnimatedText = ({
+  id,
+  children,
+  delay = 0,
+  direction = 'leftToRight',
+  customTypingSpeed,
+  ...textProps
+}: TextProps & {
+  id?: string;
+  delay?: number;
+  direction?: AnimationDirection;
+  customTypingSpeed?: number;
+}) => {
+  if (typeof children !== 'string') {
+    console.error('AnimatedText expects a string as children');
```

Comment:
do we wanna leave this here?

Reply:
removing, was copy pasta

Reply:
actually i need it, i put `children.split('')` just below to animate the characters